Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fuel-tx"
-version = "0.12.2"
+version = "0.12.3"
```

Comment:
Yeah I agree this may cause breaking behavior downstream. For instance, since we serialize this struct using serde_json, this could break deployed environments that don't have the new field specified in their chainspec.json.

Reply:
Yeah strictly speaking, its indeed a breaking change. In the VM, we will need to have a breaking anyway so we might as well set it here